src/shared/components/annotation/styles/oncokb/vars.module.scss

```diff
@@ -0,0 +1,6 @@
+$darker-blue: #064785;
```

Comment:
i think these variables will become global, so you probably want to, 1. give them domain specific names, 2. maybe place them in the global variables file